<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="ip-10-1-1-135.us-west-2.compute.internal" failures="0" tests="9" name="com.clip.tests.logistics.SmokeTest" time="0.024" errors="2" timestamp="15 Oct 2018 17:42:33 GMT" skipped="7">
  <testcase classname="com.clip.tests.logistics.SmokeTest" name="createNewOrder" time="0.000">
    <skipped/>
  </testcase> <!-- createNewOrder -->
  <testcase classname="com.clip.tests.logistics.SmokeTest" name="closeBrowser" time="0.001">
    <error message="Could not close browser java.lang.NullPointerException" type="java.lang.IllegalStateException">
      <![CDATA[java.lang.IllegalStateException: Could not close browser java.lang.NullPointerException
at com.clip.framework.MainDriver.closeBrowser(MainDriver.java:193)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
at org.testng.internal.TestMethodWorker.invokeAfterClassMethods(TestMethodWorker.java:214)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
at java.base/java.lang.Thread.run(Thread.java:844)
]]>
    </error>
  </testcase> <!-- closeBrowser -->
  <testcase classname="com.clip.tests.logistics.SmokeTest" name="login" time="0.023">
    <error message="Could not login java.lang.NullPointerException" type="java.lang.IllegalStateException">
      <![CDATA[java.lang.IllegalStateException: Could not login java.lang.NullPointerException
at com.clip.tests.logistics.SmokeTest.login(SmokeTest.java:55)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
at java.base/java.lang.Thread.run(Thread.java:844)
]]>
    </error>
  </testcase> <!-- login -->
  <testcase classname="com.clip.tests.logistics.SmokeTest" name="testCHSales" time="0.000">
    <skipped/>
  </testcase> <!-- testCHSales -->
  <testcase classname="com.clip.tests.logistics.SmokeTest" name="testCHSupervisor" time="0.000">
    <skipped/>
  </testcase> <!-- testCHSupervisor -->
  <testcase classname="com.clip.tests.logistics.SmokeTest" name="testCHRegular" time="0.000">
    <skipped/>
  </testcase> <!-- testCHRegular -->
  <testcase classname="com.clip.tests.logistics.SmokeTest" name="labelTesting" time="0.000">
    <skipped/>
  </testcase> <!-- labelTesting -->
  <testcase classname="com.clip.tests.logistics.SmokeTest" name="payByCard" time="0.000">
    <skipped/>
  </testcase> <!-- payByCard -->
  <testcase classname="com.clip.tests.logistics.SmokeTest" name="testCHMaster" time="0.000">
    <skipped/>
  </testcase> <!-- testCHMaster -->
</testsuite> <!-- com.clip.tests.logistics.SmokeTest -->
